Rakefile

```diff
@@ -50,17 +50,17 @@ namespace :crowbar do
     ).flatten.compact.uniq
   end
 
-  desc "Init all barclamps"
+  desc "Initialize all barclamps (:fork, :clone, :add_upstream, :add_susecloud)"
```

Comment:
they can also look into the Rakefile to see what it does. My point is just to enhance the documentation for others, if you look at the mailing list, the init task was apparently not well documented, and I wanted to change that.